tests/v2/validation/encryption/rke2_secret_encryption_test.go

```diff
@@ -0,0 +1,168 @@
+package encryption
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeconfig"
+	nodes1 "github.com/rancher/rancher/tests/framework/extensions/nodes"
```

Comment:
can you give this a more descriptive name

Reply:
updated it.